### DISCUSSION TITLE
Exporting Wallet private Key

### DISCUSSION THREAD

**Discussion by abstractumx** [PatrickAlphaC/nft-mix#97] [Q&A]
Hello everyone, 

I don't get the point to export the private key of the wallet (like metamask) when i set my .env file

export PRIVATE_KEY=<PRIVATE_KEY>

Why should I export it?
Can someone help me with this?
TY

**Comment by PatrickAlphaC** [comment]
How else are you going to deploy and send transactions from your code without your private key?
